src/middlewared/middlewared/plugins/account.py

```diff
@@ -15,7 +15,38 @@
 
 from dataclasses import asdict
 from middlewared.api import api_method
-from middlewared.api.current import *
+from middlewared.api.current import (
```

Comment:
Just curious, did this affect startup performance? From what I know, module is either imported or not, so there should be no difference wrt to which symbols are imported.

Reply:
No, I didn't test startup performance. Just got tired of type hints and things not working and barking at me. 😅